tests/linearGaussian_snle_test.py

```diff
@@ -3,6 +3,8 @@
 
 from __future__ import annotations
 
+from os import X_OK
```

Comment:
what is this?

Reply:
probably copilot importing `x_o` in all caps 😆